L1Trigger/Phase2L1ParticleFlow/plugins/L1MetPfProducer.cc

```diff
@@ -0,0 +1,230 @@
+#include <vector>
+#include <ap_int.h>
+#include <ap_fixed.h>
+#include <TVector2.h>
+
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DataFormats/L1TParticleFlow/interface/PFCandidate.h"
+#include "DataFormats/L1Trigger/interface/EtSum.h"
+#include "DataFormats/Math/interface/LorentzVector.h"
+
+using namespace l1t;
+
+class L1MetPfProducer : public edm::global::EDProducer<> {
```

Comment:
Should we try to harmonize the naming used across CMSSW? 
I think we use XXXMETProducer, XXXPFProducer across CMSSW. Why don't we use L1METPFProducer here?

However, I have no strong objection, just want to check with @silviodonato @qliphy